src/components/job/UpsertJob.jsx

```diff
@@ -436,6 +437,18 @@ function UpsertJobCore({ update }) {
             {...register("description", { required: true })}
           />
         </div>
+        <div className="flex flex-row items-center mb-4">
+          <label className="label flex basis-44" htmlFor="instruction">
+            Instructions of applying for the job
+          </label>
+          <textarea
+            type="text"
+            rows="4"
+            className="textarea textarea-bordered w-full"
+            placeholder="Please specify the instructions of how to apply for this job."
+            {...register("instruction", { required: true })}
+          />
+        </div>
```

Comment:
@nehakumari2307 could you help to see if this way of expression works?
i don't get what the field is about.

Reply:
This field is to provide employer a chance to specify how refugees should apply to their job posting as in email them their resume, or send their resume through online portal etc